README.md

```diff
@@ -1,9 +1,14 @@
-
-Our mobile app, HabitBuddy, provides a way for someone to start a new habit by matching them to another Buddy. Our goal is to connect Buddies with similar goals and encourage them to start a habit together. On our app, first-time Buddies fill out a basic profile and state a habit(s) they want to maintain. The Buddies will declare their new habit in the form of "habit stacking", a simple sentence format inspired by *Atomic Habits* that makes anchoring a new habit easy: After [current habit], I will [new habit]. While other habit-tracking apps boast motivating reminders and cool scheduling formats, HabitBuddy allows people to make a personal connection, not just with their habits, but with others who will inspire and motivate them. After the match, Habit Buddy provides basic contact information and lets the Buddies make the first move to connect. Because Buddies are also matched based on interests, like their favourite band or salty snack, they have another opportunity to anchor their new habit to their Buddy and their shared interest.
+Good habits are difficult to start and difficult to keep. Even the most diligent among us struggle to maintain healthy habits. According to James Clear's book, *Atomic Habits*, one of the most effective ways to start a new habit is to anchor it to an activity or interest you already do on a regular basis, which he lays out in detail through four laws: make it obvious, make it easy, make it attractive, and make it satisfying. 
+ 
+Rooted in these ideas, our mobile app, Habit Buddy, a visionary project to help you start a new habit. By matching you to someone with similar goals and interests, Habit Buddy sets you both up for success and reward by giving you the means to create accountability and encourage each other as you work towards your personal and shared goals. Habit Buddy makes it easy and fun to start and maintain a habit by following Clear’s four laws: 
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other. 
+  2. Make it easy. Habit Buddy makes your habit easy by helping you associate your new habit with something you already do and enjoy. You’ll choose an activity that you enjoy doing (i.e. drinking coffee at 8am) and commit to practicing your new habit afterward (i.e. doing yoga for 30 minutes).
+  3. Make it attractive. Habit Buddy makes your habit attractive by matching you with a Buddy who either shares similar goals and interests with you or who already practices your desired habit regularly.  
+  4. Make it satisfying. Habit Buddy makes your habit satisfying by tracking your daily progress through a streak count. Buddies can also see each other’s progress and encourage and inspire one another in practicing a new habit.
```

Comment:
Make it satisfying. **We like to repeat activities that are satisfying.** Habit Buddy makes your habit satisfying....